config/api.yml

```diff
@@ -88,6 +88,41 @@
        :post:
        - :name: create
          :identifier: alert_status_action_new
+  :alert_definition_profiles:
+    :description: Alert Definition Profiles
+    :identifier: alert_definition_profile
+    :options:
+    - :collection
+    - :subcollection
```

Comment:
I didn't see alert_definition_profiles declared as a subcollection of others, so probably drop line 96.